assets/scripts/airport.js

```diff
@@ -95,6 +98,7 @@ zlsa.atc.ArrivalDefault = Fiber.extend(function(base) {
         airline:   choose_weight(this.airlines),
         altitude:  altitude,
         heading:   this.heading,
+        fixes:     this.fixes.slice(),
```

Comment:
Should the aircraft do the clone instead of the caller?  I generally think of arguments as being immutable unless it's explicitly passed as a mutable value.